cdap-ui/app/cdap/components/Cloud/Profiles/CreateView/index.js

```diff
@@ -77,9 +85,25 @@ export default class ProfilesCreateView extends Component {
         })
       }
     };
-    console.log(`Coming soon. This is the state right now:
-      ${JSON.stringify(jsonBody, null, 2)}
-    `);
+    MyProfileApi
+      .create({
+        namespace: getCurrentNamespace(),
+        profile: this.state.profileName
+      }, jsonBody)
+      .subscribe(
+        () => {
+          this.setState({
+            redirectToNamespace: true,
+            loading: true
```

Comment:
Should this be `false` or `true`? I get that we might want to keep showing the loading state until we navigate to the namespace details page, but in that case you don't need to set `loading: true` again.